v22.2/use-cloud-storage-for-bulk-operations.md

```diff
@@ -253,12 +257,66 @@ Has permission to assume role A. See [step 2](#step-2-user). | Has a trust polic
+- [`RESTORE`](restore.html)
+
+There are multiple IAM roles referred to in this section, see the following descriptions:
+
+- _Identity role_: the IAM role you have associated with your Kubernetes service account.
+- _Operation role_: the IAM role to be assumed. This contains the permissions required to complete a CockroachDB operation.
+
+#### Set up AWS workload identity
+
+First of all, it is necessary to create an IAM role for your Kubernetes service account to assume, and then configure your CockroachDB pods to use the service account. We will refer to this IAM role as an "identity role". You can complete all of these steps with Amazon's guide on [IAM roles for service accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
+
+Once you have an identity role that your CockroachDB nodes can assume, you can configure the identity role to assume another IAM role that contains the permissions to perform a bulk operation. 
+
+1. Copy the ARN of identity role. In the Amazon management console, click on **IAM**, then **Roles**, and the name of your identity role. From the **Summary** page, copy your ARN. You will need this in the Trust Policy of the IAM role to be assumed.
+
+    <img src="{{ 'images/v22.2/aws-wi-arn-copy.png' | relative_url }}" alt="Role summary page showing the ARN copied" style="border:1px solid #eee;max-width:100%" />
+
+1. Create or open the IAM role (operation role) that your identity role will assume.
+
+    a. To create a role, click **Create Role** under the **Roles** menu. Select **Custom trust policy** and then add the ARN of your identity role to the JSON by clicking `Principal`. This will open a dialog box. Select **IAM Roles** for **Principal Type** and paste the ARN. Click **Add Principal** and then **Next**.
+
+    <img src="{{ 'images/v22.2/aws-wi-principal.png' | relative_url }}" alt="Dialog box to add principal with IAM roles selected" style="border:1px solid #eee;max-width:100%" />
+
+    b. On the **Add Permissions** page, search for the permission policies that the role will need to complete the bulk operation. Or, use the **Create Policy** button to define the required permissions. See [Storage Permissions](#storage-permissions) for detail on the minimum permissions required for each operation to complete. Click **Next**. 
+
+    c. Finally, give the role a name on the **Name, review, and create** page. The following screenshot shows the selected trust policy and permissions:
+
+    <img src="{{ 'images/v22.2/aws-wi-review-page.png' | relative_url }}" alt="Final screen in the create role process to review permissions and name role" style="border:1px solid #eee;max-width:100%" />
+
+    {{site.data.alerts.callout_info}}
+    If you already have the role that contains permissions for the bulk operation, ensure that you add the identity role ARN to the role's **Trust Relationships** tab on the **Summary** page.
+    {{site.data.alerts.end}}
```

Comment:
Question: we seem to be presenting an alternative to step 1 above, here. Step 1 also already had us at the Summary page. Is there a place higher up that this information could live? As it stands, the user needs to read through "create a new one" fully to get to "or use an existing one".